src/docs/Upcoming.mdx

```diff
@@ -0,0 +1,62 @@
+---
+name: Upcoming
+route: /upcoming
+---
+
+# Upcoming
+
+The Pablo design system is just at the beginning. We used it internally on some projects already and
```

Comment:
i would mention that both bojagi landing page and app are written in it with a link